landing-page/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>CtD class-Exerc.3-Landing page</title>
+   <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+  <header id="showcase">
+    <h1>Welcome!</h1>
+    <h1 id="simbol">Gizmatic</h1>
+        <p>Founded in 1901 Gizmatic has been making the world's best gizmos.</p> 
+    <a href="www.gizmatic.com/about" class="button1">Read More</a> 
+    
+    <section id="section-a">
```

Comment:
Looks like this section is for navigation? You could consider using the semantic [`nav` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) here instead.